### ISSUE TITLE
A method to display dynamic text to the screen

### ISSUE THREAD

**Issue by schme16** [chrismaltby/gb-studio#208]
A method to display dynamic text to the screen, with the ability to link it to a variable.

**Comment by thomas-alrek** [comment]
This is a duplicate of #146 and can be closed

**Comment by schme16** [comment]
This is a slightly different request (I'll clear it up after commenting here). I'm looking for screen overlay text, like a coin counter, or a game timer, where the text can be printed to the screen, and not just be a static background with the text burned in.

**Comment by schme16** [comment]
Hope that makes it clearer what I'm after!

**Comment by toxworks** [comment]
You can achieve this already with the current version. Using variables for value and running the counter using 'change animation frame by variable'

**Comment by schme16** [comment]
A permanently showing score counter/timer? Or just a text-box that pops up? I'm not aware of any way to do the former, but if you know how, I'd be very appreciative if you could guide me through (or just make a project that includes it, zip and upload it)

Again, for clarity I'm looking for something that could do things like this: 
![image](https://user-images.githubusercontent.com/116081/59241073-88f88380-8c49-11e9-9598-d7c67db33191.png)

**Comment by toxworks** [comment]
Yes, using actors as updatable counters is what I suggested. It's not pretty but being smart you can save some actor usage. I was able to put numbers 1-10 in a single actor, and I just update it accordingly.
